test/lib/compile/openapi.test.js

```diff
@@ -391,4 +391,32 @@ describe('OpenAPI export', () => {
         expect(openAPI.externalDocs.url).toBe('https://help.sap.com/docs/product/123.html');
   }
   );
+
+  test('OpenAPI annotations: @OpenAPI.Extensions annotation is added to the openapi document', () => {
+    const csn = cds.compile.to.csn(`
+      namespace sap.OpenAPI.test;
+      @OpenAPI.Extensions: {
```

Comment:
Additional cases for test: 

- nested annotation
- extension annotation key to start with `sap` or  `x-sap`(edge case)

For reference: https://github.com/cap-js/asyncapi/blob/main/test/lib/compile/input/valid/base.cds

Reply:
Have used the same test case to civer both these scenarios.